src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar top-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar bottom-search-bar"
+      data-qa="small"
+    >
+      <input
+        class="search-bar-input search-bar-input--bottom"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
You have two forms with the same structure and similar functionality. Consider using a semantic tag like `section` to group related content. It's recommended to use semantic tags where possible, as they improve your page's SEO and help screen readers. `div` and `span` do not have any meaning.